java/RELEASING.md

```diff
@@ -0,0 +1,102 @@
+# Releasing to Maven Central
+
+## Set up Sonatype Account
+
+* Sign up for a Sonatype JIRA account [here](https://issues.sonatype.org)
+* Click *Sign Up* in the login box, follow instructions
+
+## Get access to repository
+
+* Go to [community support](https://issues.sonatype.org/browse/OSSRH)
+* Ask for publish rights by creating an issue similar to [this one](https://issues.sonatype.org/browse/OSSRH-52782)
+  * You must be logged in to create a new issue
+  * Use the *Create* button at the top tab
+
+## Set up PGP keys
+
+* Install GNU Privacy Guard (GPG)
+  * GPG is installed by default on Ubuntu systems
+  * For other systems, see [GnuPG download page](https://www.gnupg.org/download/)
+
+* Generate the key ```gpg --gen-key```
```

Comment:
So we're signing this with the developer's personal key instead of Google's?

Reply:
We haven't set up automation on this repository, so we are not using our automation's key. This is how releases were previously done in google-cloud-java prior to automating the process.